docs/components/input.md

```diff
@@ -147,7 +147,7 @@ You can pass any input attribute and it will get properly bound to the element.
 
 ### required
 
-KInput will display an asterisk next to the label to indicate a field is required if you set the `required` attribute and provide a `label` value. See KLabel's [`required`](/components/label#required) prop for more information.
+KInput will display a red dot next to the label to indicate a field is required if you set the `required` attribute and provide a `label` value. See KLabel's [`required`](/components/label#required) prop for more information.
```

Comment:
On the KInput PR I had you change "red dot" to "asterisk"

I didn't realize it's _actually_ just a dot. Can you change "asterisk" in the KInput docs back to "red dot"?

Reply:
Already did 👍